File: 7KYU/Number of People in the Bus/solution.cpp
#include <utility>
#include <vector>

unsigned int number(const std::vector<std::pair<int, int>>& busStops){
  int bus = 0;
  
  for(auto i = 0; i < busStops.size(); i++) {
    bus = bus + busStops[i].first - busStops[i].second;
  }
  
  return bus;
}

File: 7KYU/Number of People in the Bus/solution.php
function number($bus_stops) {
  $bus = 0;
  
  for($i = 0; $i < count($bus_stops); $i++) {
    $bus = $bus + $bus_stops[$i][0] - $bus_stops[$i][1];
  }
  
  return $bus;
}